forth_src/open.fs

```diff
@@ -0,0 +1,35 @@
+\ Open a file, no error handling
+\ returns 0 on success, file # on error
```

Comment:
This comment looks wrong, should it be "\result=0 on success, or OPEN error code" ?